src/vs/workbench/parts/quickopen/browser/gotoLineHandler.ts

```diff
@@ -213,7 +215,24 @@ export class GotoLineHandler extends QuickOpenHandler {
 	}
 
 	getAriaLabel(): string {
-		return nls.localize('gotoLineHandlerAriaLabel', "Type a line number to navigate to.");
+
```

Comment:
@cleidigh this is horribly complex to read and understand, why not just:

```ts
getAriaLabel(): string {
	if (this.editorService.activeTextEditorWidget) {
		const currentLine = this.editorService.activeTextEditorWidget.getPosition().lineNumber;

		return nls.localize('gotoLineAriaLabel', "Current Line: {0}. Type a line number to navigate to.", currentLine);
	}

	return nls.localize('gotoLineHandlerAriaLabel', "Type a line number to navigate to.");
}
```

Reply:
Yeah some lack of understanding between the two quickopen approaches.  

I did not realize canRun() was already run as a base method.

For fixing #66249 I found while working on this , we need to get the aria label for the non- text editor conditions.  I have brought the same label into getAriaLabel() for that which was previously done by my extraneous canRun.

I think this is the correct logic, but I am far from understanding all of the quick open logic flow.

```ts
	getAriaLabel(): string {

		if (this.editorService.activeTextEditorWidget) {
			const currentLine = this.editorService.activeTextEditorWidget.getPosition().lineNumber;
			// Output verbose current line / go to line label
			return nls.localize('gotoLineLabelEmpty', "Current Line: {0}. Type a line number to navigate to", currentLine);
		}

		return  nls.localize('cannotRunGotoLine', "Open a text file first to go to a line");
	}
```